resource/StandardResources.md

```diff
@@ -0,0 +1,106 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus. For more details on how resources can 
+be combined see [this](Resource.md).
+
+ECS defines these fields.
+ * [Compute Unit](#compute-unit)
+   * [Container](#container)
+ * [Deployment Service](#deployment-service)
+   * [Kubernetes](#kubernetes)
+ * [Compute Instance](#compute-instance)
+   * [Host](#host)
+ * [Environment](#event)
+   * [Cloud](#cloud)
+   * [Cluster](#cluster)
+   * [Data Center](#data-center)
+
+## TODOs
+* Add logical compute units: Service, Task - instance running in a service.
+* Add more compute units: Process, Lambda Function, AppEngine unit, etc.
+* Decide if lower case strings only.
+* Consider to add optional/required for each label.
```

Comment:
Why include these in the spec instead of as an issue in the repo?

Reply:
Because I want to make sure that users understand that this is not yet completed.

Reply:
Maybe expand this node and talk about whether we will require certain combinations. Like if you started supplying `k8s` properties - you need to supply all of them

Reply:
done.